.github/workflows/issue-comment-runner.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   build:
-    if: ${{ contains(github.event.issue.title, '[build]') && contains(github.event.comment.body, '[build]') && github.event.comment.author_association == 'MEMBER' }}
+    if: ${{ contains(github.event.issue.title, '[build]') && contains(github.event.comment.body, '[build]') && (github.event.comment.author == 'mgraber' || github.event.comment.author == 'SPTKL' || github.event.comment.author == 'AmandaDoyle')}}
```

Comment:
`github.event.comment.author` ->
`github.event.comment.user.login`

https://docs.github.com/en/free-pro-team@latest/rest/reference/issues#comments

Also maybe we can try using this to make the if statement multiline
https://github.com/octokit/request-action/issues/6#issuecomment-666559952